test/index.js

```
@@ -1,42 +1,36 @@
+      mocha.run(failures => {
+        if (failures > 0) {
+          reject(new Error(`${failures} tests failed.`))
+        } else {
+          resolve()
+        }
+      })
+    } catch (err) {
+      console.error(err)
+      reject(err)
+    }
+  })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)